### DISCUSSION TITLE
How to change a specified value for a tag with multiple values

### DISCUSSION THREAD

**Discussion by sc8cn** [TriliumNext/Trilium#4187] [Q&A]
How to change a specified value for a tag with multiple values, such as changing test1 to test2.
![1](https://github.com/zadam/trilium/assets/26480118/d47eaf7a-7b84-47d8-b670-cd0aacc55075)
![2](https://github.com/zadam/trilium/assets/26480118/d855b974-8548-46db-b4a4-c412a08d1990)

**Comment by meichthys** [comment]
I might be misunderstanding your question, but you should be able to just click anywhere into the label and then modify the values in the popup:
![image](https://github.com/zadam/trilium/assets/10717998/8c5df409-211f-42f7-8d2c-0c5dec381f75)

**Reply by sc8cn** [reply]
Sorry, I didn't express myself clearly. What I mean is that I want to batch change the original values of this label.

**Reply by zadam** [reply]
You could use the "Bulk actions" (in tree context menu) to do bulk value update:

![image](https://github.com/zadam/trilium/assets/617641/77542758-13b2-4b90-a4bb-c8923af6fc09)

**Reply by sc8cn** [reply]
I tried this feature you mentioned, but it may be that I didn't understand it correctly. Now I have a label corresponding to multiple values, e.g. tag=test1, tag=test2... I just want to change test 1 to test 3. I don't understand how it should be written in the form to perform this function. Can you help demonstrate this in the form? Thank you！

**Reply by zadam** [reply]
Seems like "Update label value" from the screenshot above might be what you're looking for.
